codegen/src/lib.rs

```diff
@@ -371,6 +371,10 @@ fn default_substitutes(crate_path: &syn::Path) -> TypeSubstitutes {
             parse_quote!(sp_core::crypto::AccountId32),
             parse_quote!(#crate_path::utils::AccountId32),
         ),
+        (
+            parse_quote!(fp_account::AccountId20),
```

Comment:
Maybe add a comment that type comes from frontier which probably works for frontier/moonbeam but probably not for all ethereum-based chains.

I have no strong opinions whether to support this override or not, doesn't hurt I guess...

Reply:
Mmm it might be that we can add a few overrides if needed to cater for the common chains, but otherwise the user can do this themselves.

I was hesitent but don't see a reason not to add this since its super unlikely to ever break anything that doesn't care :)